charts/logging/loki/values.yaml

```diff
@@ -48,7 +35,7 @@ loki:
       reject_old_samples: true
       reject_old_samples_max_age: 144h
       retention_period: 720h
-    schema_config:
```

Comment:
When written _outside_ of `loki.config` (which is a templated string), this must be `schemaConfig`: https://github.com/grafana/loki/blob/v2.9.7/production/helm/loki/values.yaml#L152-L154